inlang/source-code/cross-sell/cross-sell-sherlock/src/recommendation/utils/isExtensionJson.ts

```diff
@@ -0,0 +1,10 @@
+import type { ExtensionsJson } from "./types.js"
+
+export function isExtensionsJson(object: any): object is ExtensionsJson {
```

Comment:
This is a bit hard to read, should we use a proper schema library like `valibot`? That way we could also infer the type

Reply:
done in https://github.com/opral/monorepo/pull/2360/commits/140d80395363f6c6ea7f2b80eed0cce1125b186b